### DISCUSSION TITLE
Flask-SQLAlchemy Support

### DISCUSSION THREAD

**Discussion by ConfusedCoder556** [getsentry/sentry-python#3220] [Q&A]
Is there any support for the Flask-SQLAlchemy library with sentry? I would love to monitor my queries. I've tried using the regular SQLAlchemy sentry integration but that doesn't seem to capture anything.

Has anyone else got this to work?

I'm using https://flask-sqlalchemy.palletsprojects.com/en/3.1.x/

**Comment by antonpirker** [comment]
This should work, because we patch the `sqlalchemy` package, that is also used by `flask-sqlalchemy`. 

Things to consider:
- The db queries are only available if you have tracing enabled. So please make sure you have [tracing enabled](https://docs.sentry.io/platforms/python/tracing/).
- Can you check if the SQLAlchemy integration is enabled when initializing the Sentry SDK? For this you need to set `sentry_sdk.init(..., debug=True)` and when you then start your Flask server you should see a message like ` [sentry] DEBUG: Enabling integration sqlalchemy`

if both those things are true, then you should see your queries in Sentry.io
